js/app/articleHTMLRenderer.js

```diff
@@ -266,6 +266,36 @@ const ArticleHTMLRenderer = new Knowledge.Class({
         });
     },
 
+    _get_metadata: function (model) {
+        let metadata = {
+            id: model.ekn_id,
+            title: model.title,
+            published: model.published,
+            authors: model.authors,
+            license: model.license,
+            source: model.source,
+            source_name: model.source_name,
+            source_uri: model.original_uri,
+            sets: [],
```

Comment:
because we store/provide sets (with its properties) and not just simply plain tags, so I think we should make that differentiation.